src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -1739,20 +1739,24 @@ function Test-ApplicationGatewayCRUDSubItems2
 		$appgw = Set-AzApplicationGateway -ApplicationGateway $appgw
 
 		# WAF Policy and Custom Rule
-		$variable = New-AzApplicationGatewayFirewallMatchVariable -VariableName RequestHeaders -Selector Content-Length
+        # Disabled until Firewall Policy cmdlets are updated
+		<#$variable = New-AzApplicationGatewayFirewallMatchVariable -VariableName RequestHeaders -Selector Content-Length
```

Comment:
Spoke with @number213 offline -- the commented out portions of this test will be brought back before the `network-september` branch is merged back into `master`